pkg/server/main.go

```diff
@@ -0,0 +1,74 @@
+package server
+
+import (
+	"fmt"
+	"github.com/fatih/color"
+	"github.com/k8sgpt-ai/k8sgpt/pkg/analysis"
+	"net/http"
+	"os"
+)
+
+type K8sGPTServer struct {
```

Comment:
I think renaming this to `Configuration` makes more sense as we don't have a ref to the actual `http.Serve` inside of this object

Reply:
changed this